app/views/support_tickets/start.html.erb

```diff
@@ -44,13 +44,13 @@
             <%= govuk_link_to "How and when to order laptops and tablets" , devices_how_to_order_path %>
           </li>
           <li>
-            <a class="govuk-link" href="/devices/device-allocations">
-              About device allocations
+            <a class="govuk-link" href="/devices/guide-to-resetting-windows-laptops-and-tablets">
+              How to reset Windows laptops and tablets
             </a>
           </li>
           <li>
-            <a class="govuk-link" href="/devices/device-specification">
-              Device options and specifications
+            <a class="govuk-link" href="/devices/reset-your-wireless-router">
+              How to reset routers
```

Comment:
<%= govuk_link_to "How to reset your wireless router", "/devices/reset-your-wireless-router" %>